airflow/www/templates/airflow/dags.html

```diff
@@ -36,7 +36,7 @@ <h2>DAGs</h2>
         <form id="search_form" class="form-inline" style="width: 100%; text-align: right;">
             <div id="dags_filter" class="form-group" style="width: 100%;">
               <label for="dag_query" style="width:20%; text-align: right;">Search:</label>
-              <input id="dag_query" type="text" class="typeahead form-control" data-provide="typeahead" style="width:50%;" value="{{search_query}}">
+              <input id="dag_query" type="text" class="typeahead form-control" data-provide="typeahead" style="width:50%;" value="{{search_query}}" autocomplete="off">
```

Comment:
What's your reason for removing autocomplete on this field?

Reply:
@ashb  Screenshot from Google Chrome without `autocomplete="off"`
![image](https://user-images.githubusercontent.com/1270917/60395691-f2c8c680-9b3f-11e9-810c-76993003b31c.png)

Reply:
Ah right. Makes sense!